frontend/src/components/Modal/assigndrivermodal.module.css

```diff
@@ -0,0 +1,43 @@
+.titleText {
+  margin: 16px 0px;
+  font-size: 12px;
+  font-weight: bold;
+  text-align: center;
+  color: black;
+}
+
+.driverRow {
+  height: 49px;
+  display: flex;
+  align-items: center;
+}
+
+.driverRow:hover{
+  background: rgba(240, 240, 240, 1);
+  cursor: pointer;
+}
+
+.driverName {
+  font-size: 17px;
+  color: #817E7E;
+  font-weight: bold;
+}
+
+.driverImage {
+  margin: 7px 14px 6px 27px;
+  border-radius: 50%;
+  width: 36px;
+  height: 36px;
+}
+
+.modal {
+  position: absolute;
```

Comment:
You want to make sure that this is always under the button, regardless of the screen size. Absolute positions relative to the closest positioned ancestor (https://www.w3schools.com/css/css_positioning.asp) so maybe try to give the button a position attribute so that the modal is rendered relative to its corresponding button?

Reply:
That makes a lot more sense now, sorry about that!! Fixed so that it's relative to the button and checked with different window sizes